lib/screens/login_screen.dart

```diff
@@ -180,6 +181,33 @@ class _LoginScreenState extends State<LoginScreen>
                   ),
                 ),
               ),
+              Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 25.0),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: <Widget>[
+                    GestureDetector(
+                      onTap: () {
+                        Navigator.push(
+                          context,
+                          MaterialPageRoute<dynamic>(
+                            builder: (BuildContext context) {
+                              return const ForgotPasswordScreen();
```

Comment:
I would do this with our 'standard' route (cf. routes.dart), if possible.

Reply:
Resloved 9e6a0e1